### ISSUE TITLE
Modifier keys don't work while creating touchbar button shortcut

### ISSUE THREAD

**Issue by msx2010** [folivoraAI/BetterTouchTool#2535]
**IMPORTANT:** Don't post private requests here (e.g. license issues, etc.). Please send me an email at andreas@folivora.ai instead. 

---

#### Description of bug/feature request/question:
Please only post one bug or feature request per issue.

I try to enter Shift+F5 keyboard shortcut while creating custom touch bar button, but only F5 is displayed. Ctrl, Shift, Option, Cmd key modifiers also don't work.

#### Affected input device (e.g. MacBook Trackpad, Magic Mouse/Trackpad, Touch Bar, etc.):

MacBookPro15 TouchBar

#### Device information:

Type of Mac:
macOS version: 10.13.4
BetterTouchTool version: 2.445

#### Additional information (e.g. StackTraces, related issues, screenshots, workarounds, etc.):

---

**Note:** Before bug reporting, please make sure you have the latest version of BetterTouchTool and that you have already tried to restart your system :-). If you encounter a crash, please attach a crash log from the macOS Console.app from the "User Diagnostic Reports" section.

**Comment by fifafu** [comment]
This seems to work fine here. Are you sure you are pressing all the keys at the same time?

![image](https://user-images.githubusercontent.com/8820811/39170513-28069550-479c-11e8-8697-60c3891859eb.png)

**Comment by msx2010** [comment]
Yes, this setting is active, and I tried to add BetterTouchTool to KeyCombinations->Functional Keys, but with no result.

<img width="788" alt="2018-04-24 16 25 30" src="https://user-images.githubusercontent.com/28339766/39178560-3aee8b42-47dc-11e8-904f-175ea359f154.png">

**Comment by msx2010** [comment]
You can connect me via TeamViewer, if needed.

**Comment by msx2010** [comment]
How about this issue? Can be fixed?

**Comment by msx2010** [comment]
Still not possible to create Custom Keyboard Shortcut combination with any modifiers keys. Is it possible to solve? For example I want to set Ctrl+F1 combination, but still get only F1.

**Comment by fifafu** [comment]
Unfortunately this seems to work on all machines I have tested it :-(
Maybe try using the on-screen keyboard that can be activated in System Preferences => Accessibility => Keyboard => Accessibility Keyboard, I have no idea why it doesn't work with your normal keyboard. Maybe some other software is conflicting?

**Comment by msx2010** [comment]
Modifiers keys works fine with hardware F-keys, they doesn't work only with TouchBar F-keys and only in Custom Keyboard Shortcut input box. Accessibility Keyboard works fine at that input box, but it would be great if TouchBar F-keys also work with modifiers keys in this input box, too.

**Comment by RyosukeOK** [comment]
Howdy!
I was plagued by the same condition. However, I noticed that I used the keyboard customizer application to use the US keyboard. 
I use an application called Karabiner. When I am not using the application, it will work as follows. 

<img width="1126" alt="btt_touchbar_custom_keyboard_fn_" src="https://user-images.githubusercontent.com/25970017/43988620-9dafdf72-9d74-11e8-89a3-71b2d8412ce4.png">

There is a possibility that the settings of other applications may be affected.
